src/main/java/org/broadinstitute/dsm/util/externalShipper/GBFRequestUtil.java

```diff
@@ -96,15 +97,17 @@
 
     private static Executor blindTrustEverythingExecutor;
 
-    public GBFRequestUtil() {
+    static {
+        initTrust();
+    }
+
+    private static synchronized void initTrust() {
         try {
             if (blindTrustEverythingExecutor == null) {
                 blindTrustEverythingExecutor = Executor.newInstance(SecurityUtil.buildHttpClient());
             }
-        }
-        catch (Exception e) {
+        } catch (Exception e) {
             logger.error("Starting up the blindTrustEverythingExecutor ", e);
-            System.exit(-3);
```

Comment:
This is a bit scary--a constructor that calls `System.exit()` can bring down the entire app!